keyboards/ergotravel/split_rgb.c

```diff
@@ -0,0 +1,41 @@
+#include <stdint.h>
```

Comment:
If you could remove the split rgb files?  

They're not used, and not working.

Reply:
Done